src/main/java/uk/gov/companieshouse/authcodenotification/validation/AuthCodeEmailValidator.java

```diff
@@ -13,7 +13,7 @@ public class AuthCodeEmailValidator {
 
     public static final String AUTH_CODE_REGEX = "^[A-Z0-9]{6}$";
 
-    public static final String COMPANY_NUMBER_REGEX = "^[A-Za-z]{2}[0-9]{6}$";
+    public static final String COMPANY_NUMBER_REGEX = "^[^/]([A-Za-z0-9]{6,10})$";
```

Comment:
Wondering why this has changed? Does it maybe match the reg exp in the new Oracle Query API end-point?

Reply:
It matches the check I added to the SDK https://github.com/companieshouse/private-api-sdk-java/pull/311/files

Reply:
I think it's fine - just wasn't expecting it. Makes the service more generic so I'm fine with this.